article-store/README.md

```diff
@@ -0,0 +1,130 @@
+# Article Store
+
+An XML data store for the Libero Walking skeleton.
+
+## Endpoints
+`GET`
+
+`/articles`
+
+Returns list of article ids.
+
+```{xml}
+<libero:articles xmlns:libero="http://libero.pub">
+    <libero:article>0065-1737-azm-31-03-0367</libero:article>
+    <libero:article>0034-1234-azm-32-04-0777</libero:article>
+    <libero:article>0067-1777-azm-34-08-0237</libero:article>
+</libero:articles>
+```
+
+Example:
+
+`curl -v http://localhost:8000/articles -X GET`
+
+---------------------
+
+`DELETE`
+
+`/articles/<article_id>`
+
+Deletes an entire article including all versions and content.
+
+Example:
+
+`curl -v http://localhost:8000/articles/0065 -X DELETE`
+
+---------------------
+
+`POST`
+
+`/articles/<article_id>`
```

Comment:
having a sub-resource `/articles/<article_id>/versions` to house version manipulation needs further discussion, as it follows the REST pattern of POSTing on a collection rather than on an item, and it's a published API

Reply:
Agreed. Once the decision has been made this can easily be implemented.